docs/buildkitd.toml.md

```diff
@@ -93,6 +93,7 @@ insecure-entitlements = [ "network.host", "security.insecure" ]
   enabled = true
   platforms = [ "linux/amd64", "linux/arm64" ]
   namespace = "buildkit"
+  runtime = "runc"
```

Comment:
Shouldn't this be like `io.containerd.runc.v2` ?

Reply:
Yes